### DISCUSSION TITLE
Alert component to display an error message on the UI

### DISCUSSION THREAD

**Discussion by wangchj** [heroui-inc/heroui#1212] [Help] [answered]
Hi, I really like this library. I have a question about usage. Let's say my rest API returns an error. What's the best pattern/component to use to display an error message on the UI?

Some other UI libraries have components like Alert.

**Comment by OrkhanGG** [comment]
There's no such component yet, it seems.

**Comment by winchesHe** [comment, marked as answer]
This is planned to release in v2.5.0 https://github.com/nextui-org/nextui/pull/3982
